pkg/kubelet/config/config.go

```diff
@@ -519,6 +519,7 @@ func copyPods(sourcePods []*v1.Pod) []*v1.Pod {
 		pod, err := api.Scheme.Copy(source)
 		if err != nil {
 			glog.Errorf("unable to copy pod: %v", err)
+			continue
```

Comment:
@yujuhong @mtaufen  If the err return value is not nil, the type conversion will be panic. I think this is a very serious error. We can not guarantee that err is always nil, must do err processing.